Rldx/Rldx/Rendering/DXSwapChain.h

```diff
@@ -69,7 +69,7 @@ namespace rldx
 		void Resize(ID3D11Device* device, ID3D11DeviceContext* deviceContext, UINT width, UINT height);
 		void Present(ID3D11DeviceContext* poDXDeviceContext);
 		DxTexture* GetBackBuffer() { return &m_BackBufferTexture; };
-		void UpdateViewPort(ID3D11DeviceContext* _pDeviceContext, HWND hwnWindowHandle = NULL);
```

Comment:
THis paraam "HWND" was actually meant to update/resize the current window, seem I didn't finish i

Reply:
It was unused in the two versions of the function I've seen, so I think it'll be better to remove it until a use for it is found.

Reply:
OK